build/Targets/Imports.targets

```diff
@@ -7,4 +7,5 @@
   <Import Project="..\Toolset\DefaultImports.targets"/>
   <Import Project="..\Toolset\GenerateAssemblyInfo.targets" />
   <Import Project="..\Toolset\GenerateInternalsVisibleTo.targets" />
+  <Import Project="..\Toolset\DotNetPublish.targets" />
```

Comment:
I'd prefer if this Import was directly in the EmbedIndex project rather then global for all projects.